### PR TITLE
add google and apple

### PR DESCRIPTION
**Pull request by vm**

### CODE REVIEW

File: src/views/dialogs/OnboardingDialog/ChooseWallet.tsx [hunk 1]
```diff
@@ -46,22 +47,34 @@ export const ChooseWallet = () => {
...
-          >
-            {walletType === WalletType.Discord && 'Discord'}
-            {walletType === WalletType.Email && 'Email'}
-            {walletType === WalletType.Twitter && 'Twitter'}
-            {![WalletType.Discord, WalletType.Email, WalletType.Twitter].includes(walletType) && stringGetter({ key: wallets[walletType].stringKey })}
-          </Styled.WalletButton>
-        ))}
-      </Styled.Wallets>
+      {selectedWalletType === WalletType.Email ? (
+        <div>email</div>
```
Comment (ankushswar1) [inline src/views/dialogs/OnboardingDialog/ChooseWallet.tsx:51]: This is just a placeholder right?

**Review by ankushswar1 — APPROVED** [review]
LGTM

File: src/views/dialogs/OnboardingDialog/ChooseWallet.tsx (hunk 1 above)
Comment (vm) [reply src/views/dialogs/OnboardingDialog/ChooseWallet.tsx:51]: yes

Comment (vm) [reply src/views/dialogs/OnboardingDialog/ChooseWallet.tsx:51]: i actually added this by accident, i'll rm

**Pull request merged**